torch/csrc/utils/python_arg_parser.h

```diff
@@ -127,6 +127,7 @@ struct FunctionParameter {
 
 inline at::Tensor PythonArgs::tensor(int i) {
   if (!args[i]) return at::Tensor();
+  if (args[i] == Py_None) return at::Tensor();
```

Comment:
`None` is transformed to `nullptr` in the parsing, so you shouldn't have this case.

Reply:
The root cause was that I didn't set `allow_none = True`. In this situation, a `Py_None` will get propagated here.